### PR TITLE
Introduce `send_reply` extrinsic to `gear-node-loader`

### LINKED ISSUE #1449: Implement all `pallet_gear` extrinsics generation and test
Preferably after #1448. That requires implementing:

MVP
- [x] #1601

Phase 2: 
- [ ] send_reply
- [ ] claim_value

Note: these extrinsics require some additional context to be introduced during tests generation.

### PR DESCRIPTION
**Pull request by SabaunT**
Resolves #1449 

TODO:
- [x] ~Refactor batch processing functions to be able process `send_reply` results~
- [x] Introduce `send_reply` generation into `gear-node-loader`
- [x] Introduce `claim_value` generation into `gear-node-loader`

### CODE REVIEW

**Comment by SabaunT** [comment]
Here [f2ae3af](https://github.com/gear-tech/gear/pull/2290/commits/f2ae3afe05af5ee544fb1299285129738583fa41) are "tmp" changes, far from final decision.

**Review by gshep — APPROVED** [review]
LGTM! :100:

File: gclient/src/api/storage/mod.rs [hunk 1]
```diff
@@ -34,6 +34,27 @@ use gsdk::{
 };
 
 impl GearApi {
+    /// Get up to `mails_count` messages from the mailbox.
+    pub async fn mailbox(&self, mails_count: u32) -> Result<Vec<(StoredMessage, Interval<u32>)>> {
+        self.mailbox_account(self.0.account_id(), mails_count).await
+    }
+
+    /// Get up to `mails_count` messages from the mailbox of the `account_id`.
+    pub async fn mailbox_account(
```
[the remark this answers is not in the document]
Comment (SabaunT) [reply gclient/src/api/storage/mod.rs:43]: yes, it is. It was implemented independently.
will remove that.

File: utils/node-loader/src/batch_pool.rs [hunk 2]
```diff
@@ -164,12 +166,46 @@ async fn run_batch_impl(mut api: GearApiFacade, batch: Batch) -> Result<Report>
...
             let (ex_results, block_hash) = api.create_program_batch(args).await?;
             let messages = process_ex_results(ex_results);
-            process_events(api.into_gear_api(), messages, block_hash, true).await
+            process_events(api.into_gear_api(), messages, block_hash).await
+        }
+        Batch::SendReply(args) => {
+            let removed_from_mailbox = args.clone().into_iter().map(|SendReplyArgs((mid, ..))| mid);
+
+            let (ex_results, block_hash) = api.send_reply_batch(args).await?;
+            let messages = process_ex_results(ex_results);
```
[the remark this answers is not in the document]
Comment (SabaunT) [reply utils/node-loader/src/batch_pool.rs:180]: it stands for "extrinsic results`

File: utils/call-gen/src/claim_value.rs [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+}
+
+impl From<ClaimValueArgs> for GearCall {
+    fn from(args: ClaimValueArgs) -> Self {
+        GearCall::ClaimValue(args)
+    }
+}
+
+impl TryFrom<GearCall> for ClaimValueArgs {
+    type Error = ();
```
[the remark this answers is not in the document]
Comment (SabaunT) [reply utils/call-gen/src/claim_value.rs]: done/

File: utils/node-loader/src/batch_pool.rs (hunk 2 above)
[the remark this answers is not in the document]
Comment (SabaunT) [reply utils/node-loader/src/batch_pool.rs]: changed to `extrinsic_results`

**Pull request merged**